flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/ScalarFunctionsTest.scala

```diff
@@ -1216,6 +1216,64 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
     )
   }
 
+  @Test
+  def testShiftLeft(): Unit = {
```

Comment:
Add a test case using column defined in `ScalarTypesTestBase  `, just like as follow:  
```
 testSqlApi(
      "SHIFT_LEFT(f4,1)", // f4 is BIGINT type
      "88"
    )
```